templates/pages/reset-password.jsx

```diff
@@ -87,8 +87,10 @@ var ResetPassword = React.createClass({
         queryObj = Url.parse(window.location.href, true).query;
         ga.event({category: 'Reset Password', action: 'Successfully request new password'});
         if ( queryObj.android === 'true' ) {
-          window.location = "webmaker://login?mode=sign-in";
-          return;
+          if (navigator.userAgent.toLowerCase().indexOf("android") > -1) {
```

Comment:
why not protect against breaking for people who use extensions/addons that kill off the user agent property?